### PR TITLE
docs: document process to request mana permissions

### PR DESCRIPTION
**Pull request by bynn**

### CODE REVIEW

File: docs/Project-Configuration/Project-and-Distro-Settings.md [hunk 1]
```diff
@@ -30,7 +30,15 @@ read or write (or no) access to project settings, logs, task
 annotations, and more. These can be requested for specific members or
 for entire guilds. The project admins (or Evergreen admins, if no
 project admins exist) are then able to approve/reject requested access
-for their project.
+for their project. 
+
+In order to access this functionality, you must look up the the MANA resource
```
Comment (hadjri) [inline docs/Project-Configuration/Project-and-Distro-Settings.md:35]: I'd add a mention that you have to click that little filter button in mana to look up by resource type (cause it was hard for us to see as well haha)

Comment (Kimchelly) [inline docs/Project-Configuration/Project-and-Distro-Settings.md:35]: mega nit: if this is MANA, can we also fix the paragraph above: `Mana` -> `MANA`?

Comment (Kimchelly) [inline docs/Project-Configuration/Project-and-Distro-Settings.md:35]: Can we also make it clear that the website to start requesting is the MANA website, and not Evergreen (or alternatively, just link to the place in MANA to start searching for Evergreen projects)?

**Review by Kimchelly — APPROVED** [review]
Looks good overall, just some tiny suggestions.

File: docs/Project-Configuration/Project-and-Distro-Settings.md [hunk 2]
```diff
@@ -30,7 +30,15 @@ read or write (or no) access to project settings, logs, task
...
 for entire guilds. The project admins (or Evergreen admins, if no
 project admins exist) are then able to approve/reject requested access
-for their project.
+for their project. 
+
+In order to access this functionality, you must look up the the MANA resource 
+with the type `Evergreen Project` with the name of the project that
+permissions need to be requested for. Once the project is selected, click on 
+the `Request Access` button and fill out the `Requesting for` field to see
+the specific types of permissions that can be requested on Evergreen as pictured below.
```
Comment (hadjri) [inline docs/Project-Configuration/Project-and-Distro-Settings.md:39]: Non blocking but can we update the authorization section of the dev guide saying that these values map directly to our role types + permission levels: https://docs.google.com/document/d/1KxK2-JhKiHg7ydoEJN6rAf63k94KE_5-DqlYBj48pig/edit#heading=h.grkjk8um3vob

**Review by hadjri — CHANGES_REQUESTED** [review]

File: docs/Project-Configuration/Project-and-Distro-Settings.md (hunk 1 above)
Comment (bynn) [reply docs/Project-Configuration/Project-and-Distro-Settings.md:35]: done

File: docs/Project-Configuration/Project-and-Distro-Settings.md (hunk 2 above)
Comment (bynn) [reply docs/Project-Configuration/Project-and-Distro-Settings.md:39]: added

**Review by hadjri — APPROVED** [review]

**Pull request merged**